modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/exportimport/content/processor/LayoutReferencesExportImportContentProcessor.java

```diff
@@ -941,7 +943,21 @@ else if ((content.charAt(beginPos) == CharPool.OPEN_BRACKET) ||
 			url = content.substring(beginPos + offset, endPos);
 
 			endPos = StringUtil.indexOfAny(
-				url, FriendlyURLResolverRegistryUtil.getURLSeparators());
+				url,
+				TransformUtil.transform(
+					FriendlyURLResolverRegistryUtil.getURLSeparators(),
+					s -> {
+						if (Objects.equals(
+								s,
+								VirtualLayoutConstants.
+									CANONICAL_URL_SEPARATOR)) {
+
+							return null;
+						}
+
+						return s;
+					},
+					String.class));
```

Comment:
Wouldn't it be simpler to user `List#remove`?
```
List<String> urlSeparators = FriendlyURLResolverRegistryUtil.getURLSeparators();

urlSeparators.remove(VirtualLayoutConstants.CANONICAL_URL_SEPARATOR);
```

Reply:
Yes, but getURLSeparators  returns String[] and we have to transform to a List<String>, so I think that it is more efficient to transform the array directly to another array without the CANONICAL_URL_SEPARATOR. What do you think?

Reply:
You're right :man_facepalming:  I thought it was a list. Sorry!

Reply:
But what about `ArrayUtil#remove` :wink:

Reply:
Oh yes, let's wait a moment I am changing it right now.

thanks!

Reply:
I've just changed it, thanks @adolfopa